openbas-api/src/main/java/io/openbas/migration/V3_52__Sync_archi_names.java

```diff
@@ -0,0 +1,21 @@
+package io.openbas.migration;
+
+import java.sql.Connection;
+import java.sql.Statement;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+@Component
+public class V3_52__Sync_archi_names extends BaseJavaMigration {
+
+  @Override
+  public void migrate(Context context) throws Exception {
+    Connection connection = context.getConnection();
+    Statement statement = connection.createStatement();
+    statement.execute(
+        "UPDATE payloads SET payload_execution_arch = 'x86_64' WHERE payload_execution_arch = 'X86_64';");
```

Comment:
**question:**

Why we have this changes after this migration ?
V3_51__Add_architecture_to_payload

Reply:
Because, in the previous migration, I set all values in uppercase (enum), but after reflection, it’s better to keep the values as they were before (such as x86_64, arm64), rather than introducing new values like AMD64 or ARM64.